sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/authentication/ContainerRegistryCredentialsPolicy.java

```diff
@@ -107,7 +96,7 @@ public ContainerRegistryCredentialsPolicy(TokenCredential credential, String url
      * @param tokenRequestContext the token request conext to be used for token acquisition.
      * @return a {@link Mono} containing {@link Void}
      */
-    public Mono<Boolean> authorizeRequest(HttpPipelineCallContext context, ContainerRegistryTokenRequestContext tokenRequestContext) {
```

Comment:
@g2vinay would it be possible to change the signature of authorizeRequest to return Mono<Boolean> to propogate the success\failure of authorization header?